libseccomp/src/lib.rs

```diff
@@ -1304,21 +1304,27 @@ impl ScmpFilterContext {
         ScmpAction::from_sys(ret)
     }
 
-    /// Gets the current state of the No New Privileges bit.
+    /// Gets the current state of the [`ScmpFilterAttr::CtlNnp`].
     ///
-    /// This function returns `Ok(true)` if the No New Privileges bit is set to on the filter being
+    /// This function returns `Ok(true)` if the No New Privileges flag is set to on the filter being
     /// loaded, `Ok(false)` otherwise.
     ///
     /// # Errors
     ///
     /// If this function is called with an invalid filter or an issue is
     /// encountered getting the current state, an error will be returned.
-    pub fn get_no_new_privs_bit(&self) -> Result<bool> {
+    pub fn get_ctl_nnp(&self) -> Result<bool> {
         let ret = self.get_filter_attr(ScmpFilterAttr::CtlNnp)?;
 
         Ok(ret != 0)
     }
 
+    /// Deprecated alias for [`ScmpFilterContext::get_ctl_nnp()`].
+    #[deprecated(since = "0.3.0", note = "Use ScmpFilterContext::get_ctl_nnp().")]
```

Comment:
FTR: If the next release is 0.3.0, we will need to change

https://github.com/libseccomp-rs/libseccomp-rs/blob/c9e5faa55ed6a2a0f2fc2f4c4cea261935d97487/libseccomp/src/lib.rs#L1677

Reply:
Sorry, the next release will be 0.2.3, so I fixed it.